modules/fedimint-wallet-common/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub mod tweakable;
 pub mod txoproof;
 
 pub const KIND: ModuleKind = ModuleKind::from_static_str("wallet");
-pub const MODULE_CONSENSUS_VERSION: ModuleConsensusVersion = ModuleConsensusVersion::new(2, 0);
+pub const MODULE_CONSENSUS_VERSION: ModuleConsensusVersion = ModuleConsensusVersion::new(3, 0);
```

Comment:
This is a breaking consensus change for the wallet module.

This diff will change the behavior of `process_output`, which causes a consensus failure if peers aren't on the same version.

Reply:
I don't think we need to increment it yet. We will when we actually delete the old code.

Reply:
BTW. I'd like to use this opportunity to implement consensus version changing core code to be able to fix the whole issue properly in the near future. It should be actually very simple and we have all the parts for it in already.

Reply:
https://github.com/fedimint/fedimint/blob/f98c0b18c50eaba3391d2128763e2f48dbcc5904/fedimint-core/src/module/version.rs#L86-L89

Based off the docs, I think we should increment. If peers on older versions try to process an RBF withdrawal output while peers on this version return an error, we have a breaking change to the consensus rules.

Please let me know if I'm misinterpreting the intent of module consensus versions.

Reply:
During the code review call we discussed that bumping the major version is an anti-pattern and we should instead bump the minor. Bumping the major would indicate to old clients that they're unable to communicate with servers running on that latest.